docs/About/team.md

```diff
@@ -70,6 +70,21 @@
 
     [:octicons-link-24: anneschuth.nl](https://anneschuth.nl)
 
+-   ## Christopher Spelt
+
+    Engineer
+
+    ----
+
+    ![Christopher Spelt](../img/christopher.jpeg){: .round }
+
+    After graduating in pure mathematics Christopher transitioned into machine learning. He is passionate about solving complex problems, especially those that have a societal impact. My expertise lies in math, machine learning theory and i'm skilled in Python.
```

Comment:
@ChristopherSpelt I "applied" the maximum line. If you install pre-commit (in the venv run `pre-commit install`) these type of checks will run automatically when you do the commit.

```suggestion
    After graduating in pure mathematics, Christopher transitioned into machine learning. He is passionate about
    solving complex problems, especially those that have a societal impact. My expertise lies in math, machine
    learning theory and I'm skilled in Python.
```